config/tools-manual.json

```diff
@@ -1,23 +1,267 @@
+                    "technology": ["Node js", "Markdown"],
+                    "categories": ["code-generator", "documentation-generator"]
+                }
+            },
+            {
+                "title": "Node-RED AsyncAPI plugin",
+                "description": "A plugin for generating and configuring nodes for Kafka, MQTT, AMQP, etc. automatically from an AsyncAPI specification.",
+                "links": {
+                    "repoUrl": "https://github.com/dalelane/node-red-contrib-plugin-asyncapi"
+                },
+                "filters": {
+                    "language": "HTML",
+                    "technology": ["Node-RED"],
+                    "categories": ["code-generator"]
+                }
+            },
+            {
+                "title": "MultiAPI Generator",
+                "description": "This is a plugin designed to help developers automatizing the creation of code classes from YML files based on AsyncApi and OpenAPI. It is presented in 2 flavours Maven and Gradle",
+                "links": {
+                    "repoUrl": "https://github.com/corunet/scs-multiapi-plugin"
+                },
+                "filters": {
+                    "language": "Java",
+                    "technology": ["Groovy", "Maven"],
+                    "categories": ["code-generator"]
+                }
+            }
+        ]
     },
     "Converters": {
         "description": "The following is a list of tools that do not yet belong to any specific category but are also useful for the community.",
-        "toolsList": []
+        "toolsList": [
+            {
+                "title": "Converter",
+                "description": "Converts old versions of AsyncAPI files into the latest version.",
+                "links": {
+                    "repoUrl": "https://github.com/asyncapi/converter-js"
+                },
+                "filters": {
+                    "language": "TypeScript",
+                    "technology": ["Node js"],
+                    "categories": ["converter"]
+                }
+            },
+            {
+                "title": "Converter-Go",
+                "description": "The AsyncAPI Converter converts AsyncAPI documents from versions 1.0.0, 1.1.0 and 1.2.0 to version 2.0.0. It supports both json and yaml formats on input and output. By default, the AsyncAPI Converter converts a document into the json format.",
+                "links": {
+                    "repoUrl": "https://github.com/asyncapi/converter-go"
+                },
+                "filters": {
+                    "language": "Go",
+                    "categories": ["converter"]
+                }
+            },
+            {
+                "title": "AsyncAPI-format",
+                "description": "Format an AsyncAPI document by ordering, casing, formatting, and filtering fields.",
+                "links": {
+                    "repoUrl": "https://github.com/asyncapi/converter-go"
```

Comment:
I don't think it is correct link
should be https://github.com/thim81/asyncapi-format